src/ceph.in

```diff
@@ -732,10 +731,10 @@ def main():
         print('Cluster connection aborted', file=sys.stderr)
         return 1
     except rados.PermissionDeniedError as e:
-        print('Error connecting to cluster: {0}'.format(e.__class__.__name__), file=sys.stderr)
+        print(str(e), file=sys.stderr)
```

Comment:
@tchaikov thanks for the comments! yep it's intentional. instance "e" has already has the error message, the message is assembled here.
https://github.com/ceph/ceph/blob/master/src/pybind/rados/rados.pyx#L408-L422

The original code will say:
- Error connecting to cluster: Error (Error is the name of class given by e.__class__.__name__)
- If we dont remove "Error connecting to cluster: ", it will say as this:
   Error connecting to cluster: error connecting to cluster: error code xx 
As you can see, it's quite redundant, all error message has been included in the instance of "e", we can just pull the message out.